bots/bot.go

```diff
@@ -0,0 +1,36 @@
+package bot
+
+import "crypto/tls"
+
+type BotHandleFunc func(string, []byte)
+
+type Bot interface {
+	HandleRegistration(string, []byte) string
+	Run()
+	Stop()
+}
+
+func SendRequestToCONIKS(address string, msg []byte) ([]byte, error) {
+	conf := &tls.Config{
+		InsecureSkipVerify: true,
```

Comment:
But if the sysadmin set up both the server and the registration bot, I think that the chance of them having access to both TLS certs isn't low, is it?